solution/ui/regulations/css/scss/_application_settings.scss

```diff
@@ -263,6 +265,12 @@ $width-3xl: 1920px; // The largest the site container will get
     color: $secondary_indicator_text;
 }
 
+.tertiary-indicator {
+    background: $tertiary-indicator_background;
+    color: $mid-gray;
+    border: 1px solid $mid-gray;
```

Comment:
I'd add a couple more properties to `tertiary-indicator` here so that the indicator is the same size as the primary and secondary indicators.  Right now the tertiary indicator is a few pixels taller and wider than the primary and secondary indicators, mainly because the tertiary indicator has a border and the others do not.

```
box-sizing: border-box;
padding: 1px 0;
```

`border-box` takes the border into account when determining height and width.  We're only explicitly setting width, so this will made the width 42px to match the other indicators (which don't have borders).  Without this property, the 1px border adds a px of width to each side making the indicator 44px wide.

And we're just updating the padding to counteract the 1px of top and bottom border, so the tertiary indicator becomes 20.5px tall, just like the other indicators.